README.md

```diff
@@ -102,7 +102,7 @@ See [CONTRIBUTING](docs/CONTRIBUTING.md).
 Please join our beta program, suggest your features, and report bugs:
 
 - [iOS Beta (TestFlight)](https://testflight.apple.com/join/lrKCl08I)
-- [Android Beta (Firebase)](https://appdistribution.firebase.dev/i/9ec3bca5e2b47373)
```

Comment:
Original link is also correct. I copied it from the Firebase UI:

<img width="1017" alt="image" src="https://user-images.githubusercontent.com/1799054/127625913-ece5893c-bdbd-4332-a616-c5caf9abfbb2.png">

(there is some mess with formatting because my browser is not IE^WChrome)